(Docs) 113-nmap: Adding a content (#1882)
kamranahmedse/developer-roadmap: content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/113-nmap.md

> # Nmap
> NMAP stands for Network Mapper is an open source tool that used for exploring and auditing security of the network such as checking firewall and scanning port.
> 
> `$ nmap [Scan Type ...] [Options] [host/ip]`  
> 
> e.g  
> `$ nmap google.com`
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://nmap.org/book/man.html'>NMAP Official Manual Book</BadgeLink>

Reviewer: ```suggestion
# NMAP

NMAP stands for Network Mapper and is an open-source tool used to explore and audit the network's security, such as checking firewalls and scanning ports.

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='blue' badgeText='Official Page' href='https://nmap.org/book/man.html'>NMAP Official Manual Book</BadgeLink>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.freecodecamp.org/news/what-is-nmap-and-how-to-use-it-a-tutorial-for-the-greatest-scanning-tool-of-all-time/'>What is Nmap and How to Use it</BadgeLink>
```
